lib/Zonemaster/CLI.pm

```diff
@@ -390,6 +390,7 @@ sub run {
     # align values
     $self->json( 1 ) if $self->json_stream;
     $self->raw( 0 ) if $self->json_translate; # deprecated
+    $self->raw( 1 ) if ( not $self->json_translate ); # deprecated
```

Comment:
Since we're always overriding `$self->raw` here, the `--raw` option is always ignored, right? It's probably better to only override `$self->raw` if `--[no-]translate-json` is actually specified.

Reply:
> Since we're always overriding $self->raw here, the --raw option is always ignored, right?

Only if you specify both options, something like `--no-json-translate --no-raw` which is an unexpected (and unsupported) behavior.

Reply:
Rereading your comment, I think I misunderstood it the first time. I know understand what you meant. `$self->json_translate` is set to `0` by default. So this condition has a flaw. Now that I see it, I'll provide an update.